client/lib/wpcom-undocumented/lib/site.js

```diff
@@ -195,6 +195,10 @@ UndocumentedSite.prototype.postCounts = function( options, callback ) {
 	return this.wpcom.req.get( '/sites/' + this._id + '/post-counts/' + type, query, callback );
 };
 
+UndocumentedSite.prototype.postTypeTaxonomies = function( postType, callback ) {
```

Comment:
@aduth we would wait to merge this PR https://github.com/Automattic/wpcom.js/pull/193 and try to avoid using wpcom-undocumented as much as possible. BTW this PR is pretty done.